docs/Get-ChildItem2.md

```diff
@@ -0,0 +1,246 @@
+---
+external help file: NTFSSecurity.dll-Help.xml
+Module Name: ntfssecurity
+online version:
+schema: 2.0.0
+---
+
+# Get-ChildItem2
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Gets the items and child items in one or more specified locations.  Locations are not subject to the 256 character path length limit.